### PR TITLE
Exclusively use Azure (public) feeds

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin -->
 <configuration>
     <packageSources>
         <clear />
-        <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+        <add key="internal_PublicPackages" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/internal_PublicPackages/nuget/v3/index.json" />
```
Comment (wtgodbe) [inline src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config:6]: Will this feed be accessible in public builds?

**Review by wtgodbe — APPROVED** [review]

File: src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config (hunk 1 above)
Comment (dougbu) [reply src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config:6]: I don't know but hope so

Comment (dougbu) [reply src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config:6]: In any case, I was able to `yarn install` things locally in this branch

Comment (BrennanConroy) [reply src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config:6]: Doubt, isn't this why SignalR/SignalR is failing for the similar change?

**Comment by BrennanConroy** [comment]
Didn't you say .npmrc doesn't work with yarn?

**Comment by dougbu** [comment]
> Didn't you say .npmrc doesn't work with yarn?

I thought it didn't but everything was fine locally.

File: src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config (hunk 1 above)
Comment (dougbu) [reply src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config:6]: It's possible this was all for naught. I'm checking offline w/ @magleaso

Comment (dougbu) [reply src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config:6]: OIC: We used <https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/> in [jquery-validation-unobtrusive](https://github.com/aspnet/jquery-validation-unobtrusive). I'll switch feeds here then fix up AspLabs and SignalR.

Comment (dougbu) [reply src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config]: done here

**Comment by dougbu** [comment]
@mmitche will we need a change like this in release/7.0❔ If yes, what about release/7.0-rc1❔

**Comment by dougbu** [comment]
I recall the current emphasis is on our default branch but want to confirm.

**Comment by BrennanConroy** [comment]
Can you explain what we need to do now with regards to npm dependencies, email is fine.

File: src/Components/CustomElements/src/js/.npmrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin
+registry=https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/
+
+always-auth=true
```
Comment (mmitche) [inline src/Components/CustomElements/src/js/.npmrc:4]: You'll want to remove this so that anonymous access still works.

**Review by mmitche — CHANGES_REQUESTED** [review]
the `always_auth` params should be removed, but otherwise looks good

**Comment by dougbu** [comment]
Thanks @mmitche, I'll remove the required auth in the next iteration here.

Separately, this PR seems to add .npmrc files to the `npm` packages we ship. That doesn't seem right. How would I change package.json files to exclude .npmrc from what we ship @javiercn @BrennanConroy❔

**Comment by dougbu** [comment]
> Separately, this PR seems to add .npmrc files to the `npm` packages we ship. That doesn't seem right. How would I change package.json files to exclude .npmrc from what we ship @javiercn @BrennanConroy❔

Never mind. Turns out all that happens is .npmrc files get copied to our node_modules/ folders when you do a local build. Final `*.tgz` files aren't changing here 😀

**Comment by dougbu** [comment]
@BrennanConroy @TanayParikh @javiercn and anyone else who knows our local `npm` references: I have been `yarn install`ing and occasionally `yarn build`ing locally. These steps added `"microsoft.aspnetcore.components.e2etest": "file:"` under `dependencies` in src/Components/test/E2ETest/package.json. Should I update this PR to include that change❔

**Comment by dougbu** [comment]
> Can you explain what we need to do now with regards to npm dependencies, email is fine.

sent

**Comment by dougbu** [comment]
@mmitche I addressed your concern about requiring auth.

@wtgodbe this PR is fairly different from what you approved.

**Review by wtgodbe — APPROVED** [review]

**Review by mmitche — APPROVED** [review]

**Pull request merged**